src/include/sof/lib/cache.h

```diff
@@ -22,4 +22,11 @@
 /* invalidate data */
 #define CACHE_INVALIDATE	1
 
+/* data cache line alignment */
+#if DCACHE_LINE_SIZE > 0
+#define PLATFORM_DCACHE_ALIGN	DCACHE_LINE_SIZE
+#else
+#define PLATFORM_DCACHE_ALIGN	4
```

Comment:
Why such a value?

Reply:
sizeof(uint32_t), but I would like to have the possibility to include cache.h from assembly.

Reply:
@tlauda please provide a proper description of why this change is needed. I understand that for someone that is familiar with the code this is straightforward. Anyhow, it help help reviewers to rapidly understand your code. Also, please mention in the commit message that you prefer to define PLATFORM_DCACHE_ALIGN to 4 because of including it in assembly files.

Reply:
Wouldn't something like sizeof(void*) be better than specific value?